snapcraft/plugins/maven.py

```diff
@@ -42,6 +43,26 @@
 logger = logging.getLogger(__name__)
 
 
+_MVN_SETTINGS = (
```

Comment:
A nit, I find it useful to suffix the format constants so it's easier to know that they need an argument.
Like _MVN_SETTINGS_FORMAT.